SpatialInputController.cs

```diff
@@ -182,22 +182,22 @@ void InitializeSpatialInput()
 
                     if (h.SmoothedHandFrame.Origin != Vector3f.Zero) {
                         Vector3 new_origin = 
-                            Vector3.Lerp(h.SmoothedHandFrame.Origin, h.AbsoluteHandFrame.Origin, fPositionT);
+                            Vector3f.Lerp(h.SmoothedHandFrame.Origin, h.AbsoluteHandFrame.Origin, fPositionT).ToVector3();
                         Quaternion new_rotation =
-                            Quaternion.Slerp(h.SmoothedHandFrame.Rotation, h.AbsoluteHandFrame.Rotation, fRotationT);
-                        h.SmoothedHandFrame = new Frame3f(new_origin, new_rotation);
+                            Quaternionf.Slerp(h.SmoothedHandFrame.Rotation, h.AbsoluteHandFrame.Rotation, fRotationT).ToQuaternion();
```

Comment:
No need to cast `new_origin` and `new_rotation`  to Unity types, they are converted to g3 types while creating `SmoothedHandFrame`